neo4j-jdbc/src/main/java/org/neo4j/jdbc/values/package-info.java

```diff
@@ -16,4 +16,8 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
+/**
+ * This package contains the Neo4j value systems and is public API.
```

Comment:
```suggestion
 * This package contains the Neo4j value system and is public API.
```
I assume this was meant to be singular. 🤔